package.json

```diff
@@ -86,6 +86,8 @@
     "@ethersphere/swarm-cid": "^0.1.0",
     "nanoid": "^3.1.22",
     "react": "^17.0.1",
-    "react-dom": "^17.0.1"
+    "react-dom": "^17.0.1",
+    "react-feather": "^2.0.9",
+    "react-jss": "^10.9.0"
```

Comment:
should install `npm i @types/react-jss --save-dev` but that does not work well for me locally as I have to have different puppeteer version installed locally which get propagated to packaga-lock.json